src/Microsoft.Azure.WebJobs.Extensions.EventHubs/EventHubExtensionConfigProvider.cs

```diff
@@ -17,7 +17,7 @@
 namespace Microsoft.Azure.WebJobs.EventHubs
 {
     [Extension("EventHubs")]
```

Comment:
The current config name for EventHubs extension in v1 is "eventHub" singular. I think we want it to be plural.

Reply:
Why are all the type names singular?